core/boostrenderer.py

```diff
@@ -17,6 +18,19 @@
 logger = structlog.get_logger()
 
 
+def get_body_from_html(html_string: str):
+    """Use BeautifulSoup to get the body content from an HTML document, without
+    the <body> tag."""
+    soup = BeautifulSoup(html_string, "html.parser")
+    body = soup.find("body")
+    body_content = ""
+    if body:
+        for tag in body:
+            body_content += str(tag)
+
+    return body_content
```

Comment:
@vinniefalco Because that returns the content inside the `<body>` tag, which we don't want. Our base template already renders the `<body>` tag (https://github.com/cppalliance/temp-site/blob/develop/templates/base.html#L296) so I want to strip that part.

Reply:
But if you append each tag in "body" to a string, won't the result be equal to the content inside the body tag?

Reply:
@vinniefalco I went ahead and replaced the loop with a more straightforward `replace` so it's a bit clearer what's happening.

Reply:
I wasn't complaining.. just trying to understand.